web/impact/impact/tests/test_calendar_reminder_view.py

```diff
@@ -0,0 +1,96 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+from django.contrib.auth import get_user_model
+from rest_framework.test import APIClient
+from add2cal import Add2Cal
+from impact.tests.api_test_case import APITestCase
+
+from impact.tests.factories import UserFactory
+
+User = get_user_model()  # pylint: disable=invalid-name
+
+
+class TestCalendarReminderView(APITestCase):
+    client_class = APIClient
+    user_factory = UserFactory
+
+    def setUp(self):
+        self.add2cal = Add2Cal(
+            start='20190523T144641',
+            end='20190523T144641',
+            title="mentoring session with dj shank",
+            location='boston',
+            description='test'
+        )
+
+    def test_outlook_reminder_link(self):
+        with self.login(email=self.basic_user().email):
+            params = {
+                "link_type": "outlook",
+                "start": "20190523T144641",
+                "end": "20190523T144641",
+                "title": "mentoring session with dj shank",
+                'location': 'boston',
+                'description': 'test'
+            }
+            response = self.get("calendar_reminder_view", data=params)
+            self.response_302()
+        self.assertEquals(
+            response.url,
+            self.add2cal.as_dict()['outlook_link']
+        )
+        self.assertEquals(302, response.status_code)
```

Comment:
Isn't this what `self.response_302` does? (same comment applies below)

Reply:
good call. removed